app/src/main/java/org/mozilla/fenix/components/metrics/Event.kt

```diff
@@ -319,11 +319,20 @@ sealed class Event {
             get() = hashMapOf(Events.appOpenedAllStartupKeys.source to source.name)
     }
 
-    data class AppOpenedAllSourceStartup(val source: Source) : Event() {
-        enum class Source { APP_ICON, LINK, CUSTOM_TAB, UNKNOWN }
+    data class AppStartupSourceAndType(
```

Comment:
nit: I think we can just call it `AppStartup` or similar (whatever name that doesn't clash with the cold startup probe – maybe we should rename that one). This convention is not really sustainable – what happens when we add timing values to it too? Why isn't instanceState represented?

Reply:
done